.travis.yml

```diff
@@ -27,7 +30,7 @@ before_cache:
   - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete
 cache:
   directories:
-    - $HOME/.sbt/0.13/dependency
+    - $HOME/.sbt/1.0/dependency
```

Comment:
sbt 1.x does not use this directory

Reply:
Thanks for the pickup, I will fix it tonight